CHANGES.md

```diff
@@ -1,3 +1,8 @@
+2020-03-15, Version 0.4.2
```

Comment:
2021 :laughing: :laughing: :laughing: :laughing: (we all feel like last year didn't exist)

Reply:
😆 - fixed